k8s/deploy.sh

```diff
@@ -35,12 +35,10 @@ fi
 
 helm repo add kubeshop https://kubeshop.github.io/helm-charts
 helm repo update
-helm upgrade --install $NAME kubeshop/tracetest \
+helm upgrade --install tracetest kubeshop/tracetest \
   --namespace $NAME --create-namespace \
   --set image.tag=$TAG \
-  --set image.pullPolicy=Always \
-  --set service.port=11633 \
-  --set server.httpPort=11633 \
```

Comment:
This was also needed from when we changed the default port, which has been the default in the chart for a while